docker/1.13/Dockerfile.eia

```diff
@@ -1,22 +1,36 @@
 FROM ubuntu:16.04
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+
+ARG PIP=pip3
 ARG TFS_SHORT_VERSION
```

Comment:
please set default value and since SAGEMAKER_TFS_VERSION variable is using it, could we move this before it

Reply:
Moved.